helix-term/src/commands/typed.rs

```diff
@@ -2601,6 +2618,13 @@ pub const TYPABLE_COMMAND_LIST: &[TypableCommand] = &[
             fun: open_config,
             completer: None,
         },
+        TypableCommand {
+            name: "config-open-workspace",
+            aliases: &[],
+            doc: "Open the workspace config.toml file.",
+            fun: open_workspace_config,
+            completer: None,
```

Comment:
With #5966 merged this should become

```suggestion
            signature: CommandSignature::none(),
```

to prevent a breakage on master

Reply:
thanks! I only superficially check that this has no conflicts but this only shows up as a compiler error after merge. I will rebase on master to make sure there is nothing else wrong. I should setup a bot that automatically rebases my PRs regularly and notifies me if there are conflicts or CI fails :D